plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -688,6 +689,7 @@ void GSHacksDlg::OnInit()
 	ShowWindow(GetDlgItem(m_hWnd, IDC_ALPHAHACK), ogl ? SW_HIDE : SW_SHOW);
 	ShowWindow(GetDlgItem(m_hWnd, IDC_SAFE_FBMASK), ogl ? SW_SHOW : SW_HIDE);
 	EnableWindow(GetDlgItem(m_hWnd, IDC_TC_DEPTH), ogl);
+	EnableWindow(GetDlgItem(m_hWnd, IDC_UNSCALE_POINT_LINE), ogl);
```

Comment:
``EnableWindow(GetDlgItem(m_hWnd, IDC_UNSCALE_POINT_LINE), ogl);`` **->** EnableWindow(GetDlgItem(m_hWnd, IDC_UNSCALE_POINT_LINE), ogl &&  integer_scaling != 1);

Maybe worth disabling it for native resolutions ?